### PR TITLE
Add props from all wptide repos

### LINKED ISSUE #47: Add Props
Add props for all the people that have contributed to Tide since the start in alphabetical order. They can be found in the readme of all the [`wptide`](https://github.com/wptide) repos. Search for contributors and add them to our `README.md`.

### PR DESCRIPTION
**Pull request by DavidCramer**
- [x] All props from `readme` in all repos in `wptide`
- [x] Search for contributors that got missed and add to readme.

fixes #47

### CODE REVIEW

**Comment by DavidCramer** [comment]
@valendesigns that should do it.

File: README.md [hunk 1]
```diff
@@ -599,7 +599,19 @@ a chat service, sometimes it takes several hours for community members to respon
 — please be patient.
 
 ### Credits
-Props: [@danlouw](https://github.com/danlouw), [@jeffpaul](https://github.com/jeffpaul), [@rheinardkorf](https://github.com/rheinardkorf), [@valendesigns](https://github.com/valendesigns)
+Props:  
+[Daniel Louw (@danlouw)](https://github.com/danlouw)
```
Comment (valendesigns) [inline README.md]: Let's remove the line breaks, and also add Luke and Brendan.

**Review by valendesigns — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DavidCramer) [reply README.md]: Was going to ask. Luke had a PR that in the old repo that never merged. Meaning he did work. Will update in the morning!

**Comment by DavidCramer** [comment]
@valendesigns Changes made.

**Review by valendesigns — APPROVED** [review]

**Pull request merged**